### ISSUE TITLE
obtaining unscaled pseudotime

### ISSUE THREAD

**Issue by khanthanan** [dpeerlab/Palantir#85]
Hi, 
First, thank you very much for creating this awesome package. I'm trying to compare pseudotime across phenotypes (assuming they have the same trajectory but vary at different rates). I was wondering whether there would be an option or a way for me to get the unscaled pseudotime. Since the output comes scaled to vary from 0 to 1, I'm having difficulty making a proper comparison. 

Also, I'd like to know whether is there any specific reason to scale the pseudotime? 

I'd appreciate it if you help me with it. 

Thank you, 
Kalpana

**Comment by ManuSetty** [comment]
We unfortunately dont have this explicitly supported in the code but this is an easy thing to do. You need to disable the scaling here: https://github.com/dpeerlab/Palantir/blob/79956db130b64ad3ac441a85887a51bba1e09a12/src/palantir/presults.py#L22 to return the pseudo-time value as is. 

The only reason why we scale is for ease of interpretability.  Scaling does not impact any computation.

**Comment by khanthanan** [comment]
Hi, 
Thank you very much for getting back to me. I tried what you mentioned and managed to get unscaled pseudotime. If you don't mind, I have a couple of questions. (if you want I can create a new issue on this)
As I mentioned earlier, I'm considering two phenotypes that should have a similar trajectory. So my approach was to
1. run `sce.tl.run_palantir` (I ran palantir via scanpy) on the combined dataset
2. then divide the anndata file in to two based on phenotypes. Then run `sce.tl.palantir_results` with `ms_data='X_palantir_multiscale'` separately on phenotype specific anndata file. 

I'm just wondering whether this would ensure the results I obtain, have the same trajectory but different pseudotimes. I assumed step 1 would learn the common trajectory and step 2 would map the cells onto the learned trajectory and pseudotimes. 

I didn't dig deep into the code to see whether it makes sense yet. Just wanted to get your opinion about it. 

Thank you again for your help, 
Kalpana
